templates/repo/view_file.tmpl

```diff
@@ -38,7 +38,11 @@
 	<div class="ui attached table segment">
 		<div class="file-view {{if .IsRenderedHTML}}markdown{{else if .IsTextFile}}code-view{{end}} has-emoji">
 			{{if .IsRenderedHTML}}
-				{{if .FileContent}}{{.FileContent | Str2html}}{{end}}
+				{{if .IsMarkup}}
+					{{if .FileContent}}{{.FileContent | Str2html}}{{end}}
+				{{else}}
+					<pre>{{if .FileContent}}{{.FileContent | Str2html}}{{end}}</pre>
```

Comment:
Is rly `Str2html` needed here?
`<pre>` should be inside `{{if .FileContent}}` block.